x-pack/plugins/fleet/server/telemetry/sender.ts

```diff
@@ -92,12 +94,10 @@ export class TelemetryEventsSender {
       return;
     }
 
-    const clusterInfo = await this.fetchClusterInfo();
-
     for (const channel of Object.keys(this.queuesPerChannel)) {
       await this.sendEvents(
         await this.fetchTelemetryUrl(channel),
-        clusterInfo,
+        this.clusterInfo,
```

Comment:
Does cluster info ever change once the server starts?

Reply:
`cluster-uuid` doesn't change, `version` perhaps in the rare event of stack upgrade. It seems overkill to query clusterInfo every 30s to expect upgrades. Also I'm not sure if plugins restart in case of an upgrade, in which case `fetchClusterInfo` would be called again.

Reply:
Ah cool if those are the fields then that sounds sensible 👍 thanks for clarifying